spec/acceptance/new_class_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper_acceptance'
+
+describe 'Creating a new class' do
+  before(:all) do
+    shell_ex("#{path_to_pdk} new module foo --skip-interview")
+  end
+
+  after(:all) do
+    shell_ex('rm -rf foo')
+  end
+
+  context 'when creating the main class' do
+    describe command("cd foo && BUNDLE_GEMFILE=../Gemfile #{path_to_pdk} new class foo") do
```

Comment:
I think the binstubs do work on windows as well, you just have to explicitly run them with the ruby interpreter, so `ruby /path/to/binstub/pdk <args>`